addon/services/scheduler.js

```diff
@@ -33,7 +34,7 @@ class Queue {
   }
 }
 
-export default Service.extend({
+var Scheduler = Service.extend({
```

Comment:
Would you mind updating this to `const`?

Reply:
sure!